lib/utils/person.vue

```diff
@@ -62,7 +62,7 @@
 
 <template>
   <div class="person-item" :class="{ 'has-primary-action': hasPrimaryAction }">
-    <avatar class="person-image" :url="image" @click.stop="onClick"></avatar>
+    <avatar class="person-image" :url="image" :name="name || subtitle" @click.stop="onClick"></avatar>
```

Comment:
where is this `subtitle` property coming from?

Reply:
`subtitle` is the secondary text that's passed into `person` components: https://github.com/clay/clay-kiln/blob/master/lib/utils/person.vue#L68